build.sbt

```diff
@@ -25,7 +28,8 @@ lazy val commonSettings = commonSmlBuildSettings ++ ossPublishSettings ++ Seq(
   versionScheme := Some("semver-spec"),
   scalacOptions ++= Seq("-unchecked", "-deprecation"),
   evictionErrorLevel := Level.Info,
-  ideSkipProject := (scalaVersion.value != scala2_13),
+  addCompilerPlugin("com.olegpy" %% "better-monadic-for" % "0.3.1"),
+  //ideSkipProject := (scalaVersion.value != scala2_13),
```

Comment:
it's useful to keep this setting, as in multi-version scala builds using projectmatrix intellij gets lost

Reply:
fixed